samples/admin/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+
+services:
+  admin-ui:
+    # it's a lates version of drill ui
+    image: drill4j/admin-ui
+    # To find the version you need, follow the link  https://hub.docker.com/r/drill4j/admin-ui/tags
+    ports:
+      - 3000:8080
+    environment:
+      - REACT_APP_API_HOST=drill-admin:8090
+  admin:
+    # it's a lates version of drill backend
```

Comment:
the same

Reply:
fix as the above
![image](https://user-images.githubusercontent.com/11191554/105704304-a8781900-5f1f-11eb-8d4e-0cf66e058bdd.png)